### PR TITLE
Blog post redesign

### PR DESCRIPTION
**Pull request by DSGlem**

### CODE REVIEW

File: src/styles/blog-themes/custom.scss [hunk 1]
```diff
@@ -1,102 +1,102 @@
...
-    margin: auto;
-    max-width: 700px;
-    @include media-breakpoint-up(xxl) {
-      max-width: 900px;
-    }
-    @include media-breakpoint-down(md) {
-      max-width: 100%;
-    }
-  }
+// .post {
```
Comment (lazio) [inline src/styles/blog-themes/custom.scss:4]: @DSGlem you sure we need to comment all this?

File: src/templates/BlogPost/BlogPost.js [hunk 2]
```diff
@@ -16,34 +16,54 @@ import styles from './BlogPost.module.scss';
...
+  }, []);
   return (
     <Providers>
       <BackgroundStars />
-      <Layout>
+      <Layout headerIsWhite={headerIsWhite}>
         <Head>
-          <title>{data.title} - Halo Lab Blog</title>
-        </Head>
+          <title> {data.title} - Halo Lab Blog </title>{' '}
```
Comment (lazio) [inline src/templates/BlogPost/BlogPost.js:51]: why do we need spaces in this file? `{' '}`

File: src/templates/BlogPost/BlogPost.js [hunk 3]
```diff
@@ -16,34 +16,54 @@ import styles from './BlogPost.module.scss';
 const BlogPost = ({ pageContext }) => {
   const {
     data,
+    recommendedPosts,
     recent: { previous, next },
   } = pageContext;
+
+  //selection of recommended posts in the current post
+  const RECOMMENDED_POSTS_LIMIT = 3;
```
Comment (lazio) [inline src/templates/BlogPost/BlogPost.js:24]: this code is unreadable at the moment. Can we move filter functionality to a function?

Comment (DSGlem) [reply src/templates/BlogPost/BlogPost.js:24]: fixed - https://github.com/Halo-Lab/halo-lab/pull/104

File: src/templates/BlogPost/BlogPost.js (hunk 2 above)
Comment (DSGlem) [reply src/templates/BlogPost/BlogPost.js:51]: fixed - https://github.com/Halo-Lab/halo-lab/pull/104

**Pull request merged**